### PR TITLE
rozanagy-FTS-21

### PR DESCRIPTION
**Pull request by rozanagy**
fix(levelup) update build.gradle file with necessary dependency
chore(levelup): change ddl-auto to update after the first run
refactor(levelup): remove the logic from the BadgeLibraryController
feat(levelup): create method for Badge List creation

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -39,5 +39,6 @@ dependencies {
     compile 'org.webjars:bootstrap:4.3.1'
     compile 'org.webjars:jquery:3.3.1'
     compile 'org.springframework.security.oauth.boot:spring-security-oauth2-autoconfigure:2.1.6.RELEASE'
-    compile 'org.hibernate:hibernate-core:4.3.6.Final'
+    compile 'org.hibernate:hibernate-entitymanager:5.4.2.Final'
+    compile 'org.hibernate:hibernate-core:5.2.1.Final'
 }
```
Comment (hulfyback) [inline build.gradle:44]: :heavy_check_mark:

File: src/main/java/com/greenfoxacademy/levelup/controller/BadgeLibraryController.java [hunk 2]
```diff
@@ -1,32 +1,16 @@
...
-    List<Badge> badges = new ArrayList<>();
-    badges.add(badge1);
-    badges.add(badge2);
-    badges.add(badge3);
-    badges.add(badge4);
-    model.addAttribute("badges", badges);
+    model.addAttribute("badges", Util.createListOfBadges());
     return "badgeLibrary";
   }
 }
```
Comment (hulfyback) [inline src/main/java/com/greenfoxacademy/levelup/controller/BadgeLibraryController.java:16]: :heavy_check_mark:

File: src/main/java/com/greenfoxacademy/levelup/utility/Util.java [hunk 3]
```diff
@@ -67,4 +72,18 @@ private static boolean hasNullField(Object object) throws IllegalAccessException
...
+    Badge badge3 = new Badge("3.0", "Best Colleague", "general", levels);
+    Badge badge4 = new Badge("4.0", "Lame Jokes", "general", levels);
+    List<Badge> badges = new ArrayList<>();
+    badges.add(badge1);
+    badges.add(badge2);
+    badges.add(badge3);
+    badges.add(badge4);
+    return badges;
+  }
 }
```
Comment (hulfyback) [inline src/main/java/com/greenfoxacademy/levelup/utility/Util.java:89]: :heavy_check_mark:

File: src/main/resources/application-dev.yml [hunk 4]
```diff
@@ -6,7 +6,7 @@ spring:
     driver-class-name: org.h2.Driver
   jpa:
     hibernate:
-      ddl-auto: create-drop
+      ddl-auto: update
     show-sql: true
     properties:
       hibernate:
```
Comment (hulfyback) [inline src/main/resources/application-dev.yml:12]: :heavy_check_mark:

**Review by hulfyback — APPROVED** [review]
well done!

**Review by drexeverest — APPROVED** [review]

**Pull request merged**